pennylane/operation.py

```diff
@@ -662,8 +663,8 @@ def compute_decomposition(theta, wires):
         the ``_check_batching`` method, by comparing the shape of the input data to
         the expected shape. Therefore, it is necessary to call ``_check_batching`` on
         any new input parameters passed to the operator. By default, any class inheriting
-        from :class:`~.operation.Operator` will do so within its ``__init__`` method, and
-        other objects may do so when updating the data of the ``Operator``.
+        from :class:`~.operation.Operator` will do so the first time an Operator's
+        ``batch_size`` property is accessed.
 
         ``_check_batching`` modifies the following class attributes:
```

Comment:
Actually, are they really class  attributes, or rather instance attributes? :thinking:

Reply:
updated to "instance"